src/patternfly/components/Table/table.scss

```diff
@@ -552,6 +552,14 @@
     --pf-c-table--cell--Overflow: hidden;
     --pf-c-table--cell--TextOverflow: ellipsis;
     --pf-c-table--cell--WhiteSpace: nowrap;
+
+    // stylelint-disable selector-class-pattern
+    // align icon middle when truncated
+    > .pficon,
+    > .fas {
+      vertical-align: middle;
```

Comment:
Do you think we can make this assumption? And could it move icons someone might be using today in a way they wouldn't expect? Also this won't do anything with react SVGs (which are likely the most common use case) or another icon a user might be using.

Could we remove the icons from our example, and leave sorting this out with custom CSS on the user?